tiktok_reporter_analysis/classify_reported.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import random
+from tempfile import NamedTemporaryFile
+
+import whisper
+from moviepy.editor import VideoFileClip
+from PIL import Image
+
+from .common import multi_modal_analysis
+
+
+def extract_frames_n_audio(video_path, audio_path, num_frames=2):
+    clip = VideoFileClip(video_path)
+    audio = clip.audio
+    audio.write_audiofile(audio_path)
+    duration = clip.duration
+    frames = {}
+
+    for i in range(num_frames):
+        time = random.uniform(0, duration)
```

Comment:
We should make sure we don't grab the same frame twice.  It also might be good to make sure the frames aren't too close together, but that could be an optimization for later if you prefer.